src/core/component.ts

```diff
@@ -234,7 +234,9 @@ const BaseComponent: VueConstructor<IBaseComponent> = Vue.extend({
 function cleanWidgetNode(node: Node) {
     forEachChildNode(node, (childNode) => {
         const parent = childNode.parentNode;
-        if (childNode.nodeName === "#comment" && parent) {
+        const vueComponent = (childNode as any).__vue__;
```

Comment:
It looks like the `__vue__` field is a dev-helper. Maybe it's better to mark the extension's node somehow [here](https://github.com/DevExpress/devextreme-vue/blob/master/src/core/extension-component.ts)?